### DISCUSSION TITLE
What is this window and what is it expecting me to do?

### DISCUSSION THREAD

**Discussion by wpostma** [gitextensions/gitextensions#11835] [General]
I did a pull from my github repo's upstream  fork (the developer main repo) and after I think it might be asking me for a merge commit message but (a) it doesn't say anything about a merge commit message, and (b) I can't type in this window, and I can't click OKAY.

What is this window with the cryptic title Process( [[local folder Name]])

![image](https://github.com/user-attachments/assets/06bdc342-a066-4db3-b01e-10dbd4425c4f)

I suspect it's supposed to be output window but the output is messed up and contains only two letters (zz)

**Comment by RussKie** [comment]
Yes, that's the progress dialog. Looks like something is right with your installation.
When you pull/push/fetch the same progress dialog should appear - does it behave normally (i.e., show the git command and the progress) or behave the same as in the screenshot?
